src/css/controls/imports/autostartmute.less

```diff
@@ -2,12 +2,13 @@
 
 .jw-autostart-mute {
     position: absolute;
-    bottom: 0.5em;
-    right: 0.5em;
+    bottom: 0;
+    right: 0;
```

Comment:
I don't really like that the button is flushed left button. I think you should apply the same padding that buttons in the controlbar have for consistency.